denylist/src/denylist.rs

```diff
@@ -0,0 +1,142 @@
+use crate::{client::DenyListClient, models::metadata::Asset, Error, Result};
+use bytes::Buf;
+use helium_crypto::{PublicKey, Verify};
+use serde::Serialize;
+use std::{fs, hash::Hasher, path, str::FromStr};
+use twox_hash::XxHash64;
+use xorf::{Filter as XorFilter, Xor32};
+
+pub const SERIAL_SIZE: usize = 32;
+
+/// the pubkey used to verify the signature of denylist updates
+// TODO: is there a better home for this key ?
```

Comment:
Still have a TODO here

Reply:
...oh just have to remove the TODO.  Its all good now.